karrot/groups/models.py

```diff
@@ -63,8 +57,15 @@ def annotate_yesterdays_member_count(self):
         )
 
 
+class GroupManager(models.Manager):
```

Comment:
I just noticed the _Karrot way_ seems to be to inherit from `BaseManager.from_queryset(GroupQuerySet)`. Then you can directly specify `objects = GroupManager()` in the model. Have a look at `UserManager`, `ConversationMessageManager` or `NotificationManager` for reference.

Reply:
Alright!

Reply:
Is it okay if I mark stuff as resolved or do you want to do it?